11.vpn/030.adguard-vpn-for-ios/010.features-overview/docs.en.md

```diff
@@ -5,93 +5,109 @@ taxonomy:
+Another useful feature is the import/export of exclusion lists. You can do it by following the next four steps:
 
-1. Open AdGuard VPN on the device/in the browser from where you want to export your lists of exclusions. Find the appropriate section and click on the "Export" button. The exclusions.zip archive will be downloaded.
+1. Open AdGuard VPN on the device/in the browser from where you want to export your lists of exclusions. Find the appropriate section and click on the *Export* button. The `exclusions.zip` archive will be downloaded.
 
-2. There are two TXT files inside the archive, each for Regular and Selective lists. Add more exclusions to them, delete the existing ones, rename files (but more on this later), or just leave the archive with files as it is.
+2. There are two `.txt` files inside the archive, one for each of Regular and Selective lists. Add more exclusions to them, delete the existing ones, rename files (but more on this later), or just leave the archive with files as is.
 
-3. When transferring between different devices, don't forget to send the zip file to the device for import. For example, if you import exclusion lists from your Android device to your PC, make sure to send the zip file to your PC beforehand.
+3. When transferring between different devices, don't forget to send the `.zip` file to the device for import. For example, if you import exclusion lists from your iOS device to your Mac, make sure to send the `.zip` file to your Mac beforehand.
 
-4. Open AdGuard VPN on the device/in the browser where you want to export the archive with the ready lists of exclusions. Find the appropriate section, click on the "Import" button and select the archive.
+4. Open AdGuard VPN on the device where you want to export the archive with the ready lists of exclusions. Find the appropriate section, click on the *Import* button and select the archive.
 
 SCREENSHOTS!
 
-<a name="additional-customization"></a>
+<a name="settings"></a>
+
+## App settings
+
+You can find App settings by pressing the *Settings* button in the bottom right of the screen. You can configure AdGuard VPN for iOS as you need by selecting different options.
+
+### Operating mode
+
+You can use the General mode with AdGuard VPN own protocol which provides the best combination of speed and security. While VPN usually can't interact with other VPN and ad blocking apps, the Integrated mode solves this problem and allows AdGuard VPN and AdGuard ad blocker to work together by using IPSec protocol. But there are some restrictions in this mode: you can't use the exclusion feature and choose a DNS server.
```

Comment:
While VPN usually can't interact with other VPN and ad blocking apps — > While VPN apps usually can't interact with other VPN and ad blocking apps